package.json

```diff
@@ -89,11 +89,11 @@
     "ts-node": "8.3.0",
     "tslint": "5.20.0",
     "typescript": "3.4.5",
-    "zone.js": "0.10.0"
+    "zone.js": "0.10.2"
   },
   "peerDependencies": {
-    "@angular/compiler": ">=2.3.1 <9.0.0 || >8.0.0-beta <9.0.0 || >8.1.0-beta <9.0.0 || >8.2.0-beta <9.0.0",
-    "@angular/core": ">=2.3.1 <9.0.0 || >8.0.0-beta <9.0.0 || >8.1.0-beta <9.0.0 || >8.2.0-beta <9.0.0",
+    "@angular/compiler": ">=2.3.1 <10.0.0 || >9.0.0-beta <10.0.0 || >9.1.0-beta <10.0.0 || >9.2.0-beta <10.0.0",
```

Comment:
should not be 9.0.0-next instance of 9.0.0-beta ?

Reply:
It'll match next as well because of the lexicographical ordering.